qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -189,19 +150,21 @@ def randomized_benchmarking_seq(nseeds=1, length_vector=None,
         seed_offset: What to start the seeds at (e.g. if we
         want to add more seeds later)
         align_cliffs: If true adds a barrier across all qubits in rb_pattern
-        after each set of cliffords (note: aligns after each increment
-        of cliffords including the length multiplier so if the multiplier
-        is [1,3] it will barrier after 1 clifford for the first pattern
-        and 3 for the second)
-        interleaved_gates: A list of gates of Clifford elements that
+        after each set of elements, not necessarily Cliffords
+        (note: aligns after each increment of elements including the
+        length multiplier so if the multiplier is [1,3] it will barrier
+        after 1 element for the first pattern and 3 for the second).
+        interleaved_gates: A list of gates of elements that
         will be interleaved (for interleaved randomized benchmarking)
         The length of the list would equal the length of the rb_pattern.
         is_purity: True only for purity rb (default is False)
+        group_gates: On which group (or gate set) we perform RB
+        (default is the Clifford group)
 
     Returns:
```

Comment:
I think this got out of date, can we fix it? It doesn't return the rb_opts_dict and the other outputs are conditional on the type of RB

Reply:
Done.
- rb_opts_dict deleted
- added conditions to other outputs. Note that also in case of other groups (not only cliffords) there is still an option to do interleaved RB or purity RB if needed